packages/@webex/react-component-adaptive-card/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@webex/react-component-adaptive-card",
+  "description": "Webex React AdaptiveCard",
+  "main": "./cjs/index.js",
+  "src": "./src/index.js",
+  "module": "./es/index.js",
+  "keywords": [],
+  "license": "MIT",
+  "repository": "https://github.com/webex/react-widgets",
+  "files": [
+    "src",
+    "dist",
+    "cjs",
+    "es"
+  ]
+}
```

Comment:
Where is this file coming from?

Reply:
<img width="297" alt="Screenshot 2024-10-10 at 11 43 34 AM" src="https://github.com/user-attachments/assets/dba901cf-98c8-42a9-a5dc-892adc13a5d8">
GitHub is not detecting that these files are renamed. That's why it looks like these are new files.